static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1114,6 +1114,82 @@ $quote-mark: 35px;
             margin: 0 auto;
         }
     }
+
+    .content__head--article {
```

Comment:
Same here, it should be possible to override what's in `content.scss` instead of copy-pasting the whole thing 🤔